cora/signal/lss.py

```diff
@@ -715,23 +796,45 @@ class BiasedLSSToMap(task.SingleTask):
     lognormal : bool
         Use a lognormal transform to guarantee that the density is always positive (i.e.
         delta > -1).
+    omega_HI_model : {'crichton2015', 'SKA', 'uniform'}
```

Comment:
2015 paper is actually Crighton, not Crichton
```suggestion
    omega_HI_model : {'crighton2015', 'SKA', 'uniform'}
```